lib/gui/pages/main/templates/main.tpl.html

```diff
@@ -1,6 +1,6 @@
 <div class="page-main row around-xs">
   <div class="col-xs" ng-controller="ImageSelectionController as image">
-    <div class="box text-center relative" os-dropzone="image.selectImage($file)">
+    <div class="box text-center relative" os-dropzone="image.selectImageByPath($file.path)">
```

Comment:
`$file` seems to be a path so it needs to be called as-is, so `$file.path` is `undefined` and results in an exception raised by some function nested in the call stack of `selectImagePath`.